src/ImageSharp/Metadata/Profiles/Exif/ExifEncodedStringHelpers.cs

```diff
@@ -61,17 +67,17 @@ public static bool TryCreate(ReadOnlySpan<byte> buffer, out EncodedString encode
         public static uint GetDataLength(EncodedString encodedString) =>
             (uint)GetEncoding(encodedString.Code).GetByteCount(encodedString.Text) + CharacterCodeBytesLength;
 
-        public static bool IsEncodedString(ExifTagValue tag)
+        public static byte[] GetData(EncodedString encodedString)
```

Comment:
This is used only by the writer yeah? If so I would return an `IDisposable` containing a pooled buffer which you could use to avoid the buffer allocation.

Reply:
directly writing to output buffer 5d50298

Reply:
Ace, thanks! I'll review this today.